README.md

```diff
@@ -74,5 +74,18 @@ It is possible for Cannon to verify freshly deployed contracts, using the follow
 `./run pnpm cannon verify infinex-multichain:VERSION -c CHAIN_ID --api-key ETHERSCAN_KEY`
 
 
+## useful commands
+ ```
+  CANNON_REGISTRY_PRIORITY=local pnpm cannon alter --chain-id 421614  --subpkg clone.infinex infinex-multichain:0.1.7@O2 mark-complete deploy.Forwarder deploy.AccountFactory invoke.AccountFactoryInitialize deploy.InitialProxyImplementation
+  ```
+   generates an IPFS url that can be upgraded from if steps are executing that shouldn't
```

Comment:
can you explain when cannon would try to execute steps that it shouldn't be?

Reply:
adressed